src/consumers/node-stream.js

```diff
@@ -16,23 +16,23 @@ function toNodeStream (renderer) {
 
   const read = () => {
     // If source is not ready, defer any reads until the promise resolves.
-    if (!sourceIsReady) { return; }
+    if (!sourceIsReady) { return false; }
 
     sourceIsReady = false;
     const pull = pullBatch(renderer, stream);
 
-    pull.then(result => {
+    return pull.then(result => {
       sourceIsReady = true;
       if (result === EXHAUSTED) {
-        stream.push(null);
+        return stream.push(null);
```

Comment:
The extra `return`s are there just to satisfy the linter. There's a rule called `consistent-return` enforced that says if one path of a function has an explicit `return` then all paths should.